target/linux/ramips/image/Makefile

```diff
@@ -5,12 +5,14 @@
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/image.mk
 
-DEVICE_VARS += LOADER_TYPE LOADER_FLASH_OFFS
+DEVICE_VARS += KERNEL_LOADADDR LOADER_TYPE LOADER_FLASH_OFFS
```

Comment:
KERNEL_LOADADDR is already in DEFAULT_DEVICE_VARS, so does not need to be specified here again.

Reply:
Done. Thanks.